integration-tests/vertx-http/src/test/java/io/quarkus/it/vertx/StaticResourcesTest.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.it.vertx;
+
+import static io.restassured.RestAssured.when;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class StaticResourcesTest {
+
+    @Test
+    public void testExisting() {
+        when().get("/test.txt").then().statusCode(200);
+    }
+
+    @Test
+    public void testNonExisting() {
+        when().get("/test2.txt").then().statusCode(404);
+    }
+
+    @Test
+    public void testIndexInDirectory() {
+        when().get("/dummy/").then().statusCode(200);
+    }
+
+    @Test
+    public void testNonIndexInDirectory() {
+        when().get("/dummy2/").then().statusCode(404);
+    }
+
+    @Test
+    public void testIndexInNonExistingDirectory() {
+        when().get("/dummy2/").then().statusCode(404);
```

Comment:
I guess this should be something like

```suggestion
        when().get("/dummy3/").then().statusCode(404);
```

Reply:
Done